### PR TITLE
add new guidance on the use of GitHub pages for static content

### PR DESCRIPTION
**Pull request by pauldougan**
- new section based on the paas tech docs that migrated to github pages
- updated the updates section

# who to review 
anyone but @pauldougan 

# how to review
- clone branch
- npm install && npm run local
- https://localhost:3000
- ensure content is readable (the target audience is a team with static docs based on TDT looking for help migrating their own content)
- ensure the links go somewhere sensible

### CODE REVIEW

File: pages/content/migration-guidance.mdx [hunk 1]
```diff
@@ -6,9 +6,10 @@ title: "Migrating from GOV.UK PaaS"
 
 ## Last updated
 
-- 18 May 2023 - added new guidance ['Finalise your GOV.UK PaaS account'](#finalise-your-govuk-paas-account) 
-- 5 Jan 2023 - added new guidance ['PaaS CDB Migration'](#paas-cdn-migration) 
-- 2 November 2022 - first published
+- 2023-08-15 - new guidance section ['Publishing static pages to GitHub Pages'](#publishing-static-pages-to-github-pages)
```
Comment (kr8n3r) [inline pages/content/migration-guidance.mdx:9]: let's use a human-readible date format as before

File: pages/content/migration-guidance.mdx [hunk 2]
```diff
@@ -373,6 +374,56 @@ Principles](https://www.ncsc.gov.uk/collection/cloud/the-cloud-security-principl
 - Use mature, well documented technologies and that you can readily get skills for in the marketplace
 - Consider running some technical spikes to evaluate alternatives and to inform your decision
 
+### Publishing static pages to GitHub Pages
+
+The [GDS technical writers]() provide a commonly used tool to generate static documentation from templates called the [Tech Doc Templates](https://tdt-documentation.london.cloudapps.digital/).
```
Comment (kr8n3r) [inline pages/content/migration-guidance.mdx:379]: missing link

File: pages/content/migration-guidance.mdx [hunk 3]
```diff
@@ -373,6 +374,56 @@ Principles](https://www.ncsc.gov.uk/collection/cloud/the-cloud-security-principl
...
+- set up Nodejs tools using [actions/setup-node](https://github.com/marketplace/actions/setup-node-js-environment)
+- set up Ruby tools using [actions/setup-ruby](https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby)
+- set up Python tools using [actions/setup-python](https://github.com/marketplace/actions/setup-python)
+- install Python and Nodejs dependencies
+- run tests in Nodejs
+- run tests in Python
+- run Middleman using `bundle exec middleman build` to generate static content in the `/build` directory
+- create an artifact from the generated `/build` directory using the [actions/upload-github-pages-artifact](https://github.com/marketplace/actions/upload-github-pages-artifact) storing the content in a gzip archive called `github-pages`
+- publish to GitHub pages using the [actions/deploy-pages](https://github.com/actions/deploy-pages)
+- notify the PaaS team via Slack if there are any problems
```
Comment (kr8n3r) [inline pages/content/migration-guidance.mdx:417]: do we need to explain what each action does?

File: pages/content/migration-guidance.mdx (hunk 1 above)
Comment (pauldougan) [reply pages/content/migration-guidance.mdx:9]: as a human I prefer [ISO8601 https://en.wikipedia.org/wiki/ISO_8601 format dates :) however I will change the dates

File: pages/content/migration-guidance.mdx (hunk 3 above)
Comment (pauldougan) [reply pages/content/migration-guidance.mdx:417]: yes

File: pages/content/migration-guidance.mdx (hunk 2 above)
Comment (pauldougan) [reply pages/content/migration-guidance.mdx]: fixed in new commit

**Review by kr8n3r — APPROVED** [review]

**Pull request merged**